### DISCUSSION TITLE
Terminal line wrapping

### DISCUSSION THREAD

**Discussion by ghost** [Eugeny/tabby#7992] [Q&A] [answered]
I really love Tabby, but I got some issues with the line wrapping when viewing logs, have we got a checkbox to enable or disable line wrapping? or we set it somewhere in the configuration file. Thanks so much.

**Comment by Eugeny** [comment, marked as answer]
Thanks! Line wrapping isn't controller by the terminal. If you're using e.g. less to view logs, you could try less -S to disable wrapping.

**Comment by lawweiliang** [comment]
@vatisteve 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
